bazel/repositories.bzl

```diff
@@ -219,30 +233,29 @@ def envoy_dependencies(path = "@envoy_deps//", skip_com_google_protobuf = False,
     # semi-standard in the Bazel community, intended to avoid both duplicate
     # dependencies and name conflicts.
     existing_rule_keys = native.existing_rules().keys()
-    if not ("backward" in skip_targets or "com_github_bombela_backward" in existing_rule_keys):
+    if "backward" not in skip_targets:
```

Comment:
Can we have a local dict here mapping string -> instantiation and iterate over it? Might also reduce boilerplate further. Alternatively, let's kill `skip_targets`. Do we still care about this @lizan ?

Reply:
+1 for killing `skip_targets`. Skipping by repo name is much nicer.

Reply:
As long as istio/proxy builds, I'm fine with it. We do `skip_targets` for go toolchains.

Reply:
OK, I've kept `skip_targets` in place for the Go toolchain deps so Istio Proxy doesn't have to change immediately, and dropped it for the others. We can finish it off once nothing uses it.